Libs/Optimize/ParticleSystem/itkParticleGradientDescentPositionOptimizer.txx

```diff
@@ -192,11 +239,14 @@ namespace itk
                   domain->ApplyConstraints(pt);
                   m_ParticleSystem->SetPosition(pt, it.GetIndex(), dom);
 
+                  // Debuggg
+                  stream << "Not good energy " << pt << std::endl;
+                  //stream << std::endl;
+
                   m_TimeSteps[dom][k] /= factor;
                 }
                 else // keep the move with timestep 1.0 anyway
                 {
-                  if (gradmag > maxchange) maxchange = gradmag;
```

Comment:
Was there a reason for this change?  Is it related to constraints?  What's the effect?

Reply:
Nice catch! I don't know if this had any purpose, but I missed it. When I merged from master, the tracking this file was all messed up and I had to compare stuff manually. I've changed it back. It shouldn't affect the results.